### PR TITLE
feat: Add hideUnavailableItems at store.config

### PR DESCRIPTION
**Pull request by beatrizmaselli**
## What's the purpose of this pull request?

Add hideUnavailableItems at store.config

Linked to this PR: https://github.com/vtex/faststore/pull/1180

## How to test it?

Change the hideUnavailableItems config

## References

You can check the problem here: https://base.vtex.app/technology/

Total count and number of products does not match.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,13 +27,17 @@
...
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
-    "@faststore/api": "^1.6.7",
-    "@faststore/sdk": "^1.6.7",
-    "@faststore/ui": "^1.6.7",
-    "@vtex/gatsby-plugin-nginx": "^1.6.7",
-    "@vtex/gatsby-plugin-thumbor": "^1.6.7",
-    "@vtex/gatsby-source-store": "^1.6.7",
-    "@vtex/graphql-utils": "^1.6.7",
+    "@faststore/api": "https://pkg.csb.dev/vtex/faststore/commit/56eafc23/@faststore/api/_pkg.tgz",
```
Comment (tlgimenes) [inline package.json:30]: just remember to update these packages before releasing. Also, don't add unecessary packages, like `gatsby-plugi-source-vtex`  etc

**Review by anamaiabap — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (beatrizmaselli) [reply package.json]: Done!

**Review by filipewl — APPROVED** [review]
Thanks for the PR!

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add hideUnavailableItems at store.config
+  
+### Added
+
```
Comment (filipewl) [inline CHANGELOG.md:15]: There's already an "Added"  heading above so no need to repeat this one. Sorry if I wasn't clear!
```suggestion
```
I'm going to commit this suggestion, okay?

**Pull request merged**